django_coverage_plugin/plugin.py

```diff
@@ -162,6 +163,15 @@ def file_tracer(self, filename):
     def file_reporter(self, filename):
         return FileReporter(filename)
 
+    def find_unexecuted_files(self, src_dir):
+        res = []
+        for i, (dirpath, dirnames, filenames) in enumerate(os.walk(src_dir)):
+            for filename in filenames:
+                if re.match(r"^[^.#~!$@%^&*()+=,]+\.html?$", filename):
```

Comment:
Also I expect to make a comment in the coverage.py PR that find_unexcuted_files() should be a generator, so it should `yield` the matching files.